passageflex/src/main/java/id/passage/passageflex/Utils.kt

```diff
@@ -0,0 +1,20 @@
+package id.passage.passageflex
+
+import android.app.Activity
+
+internal class Utils {
```

Comment:
So these utils are meant to apply to all of PassageFlex including non-passkey flex options in the future, correct?

Reply:
Yup!